### ISSUE TITLE
Feature: add an assertion of float value

### ISSUE THREAD

**Issue by shigedangao** [itzseven/cunit#36]
Asserting float value is not currently support. 
However this can be a good idea to compare float 
As we know, float are not easy to compare so we can use a **round** way to check the float equality like below 

```c
float roundFloat(float entry, int factor) {
    return roundf(entry * factor) / factor;
}
```

**Comment by itzseven7** [comment]
Added in https://github.com/itzseven/cunit/tree/float_comparison

Used the basic condition with epsilon, keep in mind that it can fail in some cases because of float representation and rounding.

Please check on your side :)

**Comment by shigedangao** [comment]
Hi ! I'll check that tomorrow thanks for the great works 👍

**Comment by shigedangao** [comment]
Hi ! I did implement your feature and it works pretty well ! Below is my example of implementation of your feature. Please note that there are some travis test currently running. However as travis has some issue tonight the build my take sometimes. 

Test file with your decimal assertion feature: [lch test](https://github.com/MarcInthaamnouay/lymui/blob/feat/other_xyz_colors/lymui/tests/lch_test.c) 

Travis build: [Travis build](https://travis-ci.org/MarcInthaamnouay/lymui/builds/377900480?utm_source=github_status&utm_medium=notification)

Thanks, great job ! 👍

**Comment by itzseven7** [comment]
Merged in #37 

Will probably enhance the comparison in future versions :)

Thanks for the support!
